### PR TITLE
content: media archive

### PR DESCRIPTION
**Pull request by Fupete**

### CODE REVIEW

**Comment by Fupete** [comment]
Reminder: remove links to youtube auto transcriptions if not an a11y days video.

File: src/data/content/community/media/20210706-sviluppare-nuovi-servizi-pubblici-in-modalità-open-source.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          align: "middle"
+          color: "secondary"
+        label: Media
+        url: "../"
+
+components:
+  hero:
+    background: null
+    centered: true
+    title: 'Sviluppare nuovi servizi pubblici in modalità open sourc'
```
Comment (danielaiozzo) [inline src/data/content/community/media/20210706-sviluppare-nuovi-servizi-pubblici-in-modalità-open-source.yaml:34]:
```suggestion
    title: 'Sviluppare nuovi servizi pubblici in modalità open source'
```

File: src/data/content/community/media/20210615-progettare-un-servizio-pubblico-digitale.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          trascription: |
+            È disponibile la sola trascrizione generata in automatico nella [scheda del video su YouTube](https://youtube.com/watch?v=3iNuwSJs-gI)
+
+    - noSpace: false 
+      centered: true
+      components:
+        - name: TextImageCta
+          # title: 
+          text: |
+            Webinar del 15/06/2021 del corso "Progettare, sviluppare ed erogare servizi pubblici digitali". Come realizzare un servizio digitale? In questo video, vengono affrontati il processo di progettazione, di sviluppo e la messa in produzione con un foocus sulla progettazione di un servizio digitale di qualità: per esempio, cosa vuol dire progettare per l'utente un servizio pubblico digitale, come fare ad avere un impatto positivo sul cittadino e sul sistema Paese, come impostare un processo di progettazione virtuoso (principi e competenze necessarie).
```
Comment (danielaiozzo) [inline src/data/content/community/media/20210615-progettare-un-servizio-pubblico-digitale.yaml:74]:
```suggestion
            Webinar del 15/06/2021 del corso "Progettare, sviluppare ed erogare servizi pubblici digitali". Come realizzare un servizio digitale? In questo video, vengono affrontati il processo di progettazione, di sviluppo e la messa in produzione con un focus sulla progettazione di un servizio digitale di qualità: per esempio, cosa vuol dire progettare per l'utente un servizio pubblico digitale, come fare ad avere un impatto positivo sul cittadino e sul sistema Paese, come impostare un processo di progettazione virtuoso (principi e competenze necessarie).
```

File: src/data/content/community/media/20211109-progettare-il-sito-web-di-un-comune.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          trascription: |
+            È disponibile la sola trascrizione generata in automatico nella [scheda del video su YouTube](https://youtu.be/Lzx778I8toA?si=ty6tAs7UGJek5crx)
+
+    - noSpace: false 
+      centered: true
+      components:
+        - name: TextImageCta
+          # title: 
+          text: |
+          Webinar del 09/11/2021 del corso "Progettare e realizzare il sito web di un Comune". Il webinr intende fornire informazioni di contesto sulle Linee Guida di riferimento e sulla metodologia di progettazione proposta da Designers Italia. In particolare, quali sono le attività da svolgere e quali strumenti concreti Designers Italia mette a disposizione per le diverse figure coinvolte nei progetti di ideazione dei servizi pubblici digitali della PA: risorse, strumenti operativi e casi studio di buone pratiche.
```
Comment (danielaiozzo) [inline src/data/content/community/media/20211109-progettare-il-sito-web-di-un-comune.yaml:74]:
```suggestion
          Webinar del 09/11/2021 del corso "Progettare e realizzare il sito web di un Comune". Il webinar intende fornire informazioni di contesto sulle Linee Guida di riferimento e sulla metodologia di progettazione proposta da Designers Italia. In particolare, quali sono le attività da svolgere e quali strumenti concreti Designers Italia mette a disposizione per le diverse figure coinvolte nei progetti di ideazione dei servizi pubblici digitali della PA: risorse, strumenti operativi e casi studio di buone pratiche.
```

File: src/data/content/community/media/20211123-l-architettura-informativa-i-contenuti-e-l-interfaccia-utente-del-sito-web-di-un-comune.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+          trascription: |
+            È disponibile la sola trascrizione generata in automatico nella [scheda del video su YouTube](https://youtu.be/JtOHiIaOPmA?si=Y8N7GGK0NJCSSH3n)
+
+    - noSpace: false 
+      centered: true
+      components:
+        - name: TextImageCta
+          # title: 
+          text: |
+            Webinar del 23/11/2021 del corso "Progettare e realizzare il sito web di un Comune". Il Modello di sito web di un Comune propone template pronti per l'uso e un pacchetto di strumenti per aiutare ad offrire un'esperienza d'uso efficace e inclusiva agli utenti. Dove sono e come si utilizzano le risorse e gli strumenti messi a disposizione: stato dell'arte e prospettive, con uno sguardo a dove trovare la community di riferimento.
```
Comment (danielaiozzo) [inline src/data/content/community/media/20211123-l-architettura-informativa-i-contenuti-e-l-interfaccia-utente-del-sito-web-di-un-comune.yaml:74]:
```suggestion
            Webinar del 23/11/2021 del corso "Progettare e realizzare il sito web di un Comune". Il Modello di sito web di un Comune propone template pronti per l'uso e un pacchetto di strumenti per aiutare a offrire un'esperienza d'uso efficace e inclusiva agli utenti. Dove sono e come si utilizzano le risorse e gli strumenti messi a disposizione: stato dell'arte e prospettive, con uno sguardo a dove trovare la community di riferimento.
```

File: src/data/content/community/media/20211124-rilascio-e-riuso-di-un-software-open-source.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+# CREATEPAGE GATSBY
+# ————————————————————————————————————————————
+metadata:
+  template:
+    name: level3 # article?
+  archive: media
+
+seo:
+  name: "Rilascio e riuso di un software open source"
+  description: Le esperienze del Consiglio Nazioale delle Ricerche e dell’Istituto Superiore di Sanità.
```
Comment (danielaiozzo) [inline src/data/content/community/media/20211124-rilascio-e-riuso-di-un-software-open-source.yaml:11]:
```suggestion
  description: Le esperienze del Consiglio Nazionale delle Ricerche e dell’Istituto Superiore di Sanità. 
```

File: src/data/content/community/media/20211124-rilascio-e-riuso-di-un-software-open-source.yaml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+          color: "secondary"
+        label: Media
+        url: "../"
+
+components:
+  hero:
+    background: null
+    centered: true
+    title: 'Rilascio e riuso di un software open source'
+    subtitle: Le esperienze del Consiglio Nazioale delle Ricerche e dell’Istituto Superiore di Sanità
```
Comment (danielaiozzo) [inline src/data/content/community/media/20211124-rilascio-e-riuso-di-un-software-open-source.yaml:35]:
```suggestion
    subtitle: Le esperienze del Consiglio Nazionale delle Ricerche e dell’Istituto Superiore di Sanità
```

File: src/data/content/community/media/20220209-l-importanza-di-un-design-system-per-i-servizi-pubblici-digitali.yaml [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+      id: kangarooExample
+      titleSr: Metadati e link per approfondire
+      noPadding: true
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Design system, Porgettazione interfaccia, Sviluppo interfaccia]
```
Comment (danielaiozzo) [inline src/data/content/community/media/20220209-l-importanza-di-un-design-system-per-i-servizi-pubblici-digitali.yaml:50]:
```suggestion
      tags: [Design system, Progettazione interfaccia, Sviluppo interfaccia]
```

File: src/data/content/community/media/20220323-scopriamo-bootstrap-italia-2-0.yaml [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+          trascription: |
+            È disponibile la sola trascrizione generata in automatico nella [scheda del video su YouTube](https://youtube.com/watch?v=dQVy8DLPJbY)
+
+    - noSpace: false 
+      centered: true
+      components:
+        - name: TextImageCta
+          # title: 
+          text: |
+            Community lab di Developers Italia e Designers Italia del 23/03/2022. Incontro dedicato a Bootstrap Italia 2.0, una delle risorse del design system del Paese, e alla sua utilità per sviluppare le interfacce dei servizi pubblici digitali.
```
Comment (danielaiozzo) [inline src/data/content/community/media/20220323-scopriamo-bootstrap-italia-2-0.yaml:74]:
```suggestion
            Community lab di Developers Italia e Designers Italia del 23/03/2022. Incontro dedicato a Bootstrap Italia 2.0, una delle risorse del design system del Paese e alla sua utilità per sviluppare le interfacce dei servizi pubblici digitali.
```

**Review by danielaiozzo — CHANGES_REQUESTED** [review]

**Review by danielaiozzo — APPROVED** [review]

**Comment by Fupete** [comment]
I will merge this tomorrow morning.

**Pull request merged**